### ISSUE TITLE
False Positive: E1205: Too many arguments for logging format string

### ISSUE THREAD

**Issue by mahillah** [pylint-dev/pylint#9835]
### Bug description

```python
except Exception as e:
    logging.error("Error connecting to Kibana or retrieving spaces: %s", e)

# Please provide further clarification for when this error is raised.
```

### Command used

```shell
pylint config.py
```

### Pylint output

```python
script/config.py:74:8: E1205: Too many arguments for logging format string (logging-too-many-args)
```

### Expected behavior

No error since there are not too many arguments

### Pylint version

```shell
pylint 3.3.0
```

### OS / Environment

centos 7 docker image

**Comment by jacobtylerwalls** [comment]
Tested with this example, couldn't reproduce:
```py
import logging
try:
    1/0
except Exception as e:
    logging.error("Error connecting to Kibana or retrieving spaces: %s", e)
```
